### DISCUSSION TITLE
[QUESTION] Using OML2OWL with Modelica OML Ontology does not work with a OMLv1 file. How to migrate from OMLv1 to OMLv2?

### DISCUSSION THREAD

**Discussion by kataph** [opencaesar/platform#30] [Q&A] [answered]
As per title, I am trying to convert this ontology: https://github.com/OpenModelica/ModelicaOML/tree/adrpo/src/main/oml/openmodelica.org/openmodelica
from OML to OWL. 
I am trying to use the OML2OWL adapter. The adapter builds and runs, but returns errors. The errors appear to be due the fact that Modelica OML ontology is written in OMLv1, therefore the parser takes issue. Attempts to rollback the adapter to previous commits which should work with OMLv1 didn't work (build fails, but feel free to suggest how to solve this issue also if needed), therefore my decision was to migrate the modelica ontology from OMLv1 to OMLv2. So I replaced ":>" with "<", but then found "structure", which is a type from OMLv1, which has been suppressed in OMLv2. Moreover, the OMLv2 specification document, despite having a section on how to migrate from v1 to v2, ignores completely the "structure" type. 

In summary, 
how do I migrate a OMLv1 (specifically the modelica ontology) file to OMLv2, considering also the presence of the "structure" type (and possibly other things not mentioned in the migration guide)?

Kind regards and have a nice day

**Comment by melaasar** [comment, marked as answer]
I added the missing instructions for migrating `structure` in the guide (they got missed as this change occurred later in v2). Basically, replace `structure` by a `concept`, and `structured property` by `relation`. Let me know if you're still having issues.

**Reply by kataph** [reply]
Perfect, many thanks
